pkgs/by-name/az/azuki-b/package.nix

```diff
@@ -0,0 +1,32 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchzip,
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "azuki-b";
+  version = "1.20";
+  downloadVersion = "B120";
+
+  src = fetchzip {
+    url = "https://azukifont.com/font/azukifont${downloadVersion}.zip";
```

Comment:
They are all from the same source and installing the same files? If it’s not big, there’s actually no need to unpack it.